lib/customs.js

```diff
@@ -0,0 +1,29 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+var P = require('./promise')
+var Pool = require('./pool')
+
+function Customs() {
+  this.pool = new Pool("http://127.0.0.1:7000", { timeout: 1000 })
```

Comment:
need this one to use config. see how we `require` config in other modules and pull the customs url from the config.